lib/boombox/internal_bin.ex

```diff
@@ -289,6 +289,19 @@ defmodule Boombox.InternalBin do
     {[], state}
   end
 
+  @impl true
+  def handle_child_setup_completed(:udp_source, _ctx, state) do
+    {[notify_parent: :external_resource_ready], state}
+  end
+
+  def handle_child_setup_completed(:rtsp_source, _ctx, state) do
+    {[notify_parent: :external_resource_ready], state}
+  end
+
+  def handle_child_setup_completed(_child, _ctx, state) do
+    {[], state}
+  end
```

Comment:
in majority of the plugins there is a convention to write `@impl true` before each callback match